libcnb-test/Cargo.toml

```diff
@@ -20,6 +20,7 @@ fs_extra = "1.3.0"
 libcnb-common.workspace = true
 libcnb-data.workspace = true
 libcnb-package.workspace = true
+regex-lite = "0.1.5"
```

Comment:
Question - should we use this or the full `regex` crate?

Reply:
I know that clippy will check the `regex` crate for valid regex-s and that it will not check `fancy-regex`. I don't know if it will check regex-lite. Looking at the source it does not look like it would https://github.com/rust-lang/rust-clippy/issues/10205. I think we should use the `regex` crate.

Reply:
@schneems because the regex is passed to the macro as a string value I don't believe the linter will be able to check those. And I'm less concerned about having linting here because these macros are intended for testing so if an invalid regex is given, you'll end up with a panic that fails a test.

Reply:
I'm happy to swap `regex-lite` for `regex` though. For test code I don't think we need to be as concerned with compile or binary sizes.